src/openrct2-ui/windows/Park.cpp

```diff
@@ -656,7 +657,7 @@ static void window_park_entrance_mouseup(rct_window *w, rct_widgetindex widgetIn
         break;
     case WIDX_RENAME:
         set_format_arg(16, uint32, gParkNameArgs);
-        window_text_input_open(w, WIDX_RENAME, STR_PARK_NAME, STR_ENTER_PARK_NAME, gParkName, 0, 32);
+        window_text_input_open(w, WIDX_RENAME, STR_PARK_NAME, STR_ENTER_PARK_NAME, gParkName, 0, RCT12_USER_STRING_MAX_LENGTH);
```

Comment:
`RCT12_xyz` is only used in parts directly interfacing with SV6/SC6 - `USER_STRING_MAX_LENGTH` is what you want here.

Reply:
D'oh! Changed.